app/models/question.rb

```diff
@@ -18,8 +18,10 @@
 class Question < ApplicationRecord
   belongs_to :documentation
   validates :body, :answers, :correct_answers, :explanation,
```

Comment:
Thx for this one.
Don't hesitate to go a bit further, to set the validation more explicit:

```
   validates :body,
             :answers,
             :correct_answers,
             :explanation,
             :documentation_id,
             :answer_counter,
             :positive_rates,
             :negative_rates, presence: true
```